src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = 40 * days;
```

Comment:
Prefer using 'const' over 'let' for the 'result' variable since its value is not being reassigned.